package.json

```diff
@@ -60,6 +60,7 @@
     "@wordpress/hooks": "2.1.0",
     "@wordpress/i18n": "3.2.0",
     "@wordpress/plugins": "2.1.0",
-    "@wordpress/wordcount": "2.1.0"
+    "@wordpress/wordcount": "2.1.0",
+    "classnames": "^2.2.6"
```

Comment:
This should have the corresponding update to `package-lock.json`, yeah>? Also, it should be `"2.2.6"` exactly, not `"^2.2.6"`

Reply:
Hmm, I thought I committed the lock file as well.

Also, you're right with the version number. By the way while we're at: is the reasoning behind this written down somewhere?

Reply:
The reason is just that Renovate will handle updating the version numbers for us. They have good docs (which I just found) about why to do it: https://renovatebot.com/docs/dependency-pinning/#why-pin-dependencies

Reply:
Looks like the lock file was unaffected because it was already a dependency of all the WordPress packages anyway. Nothing to do there.